### PR TITLE
fix newline in install docs

### PR DESCRIPTION
**Pull request by trastle**
make echo obey the newline in the install snippet

### CODE REVIEW

**Comment by mark-rushakoff** [comment]
Looks good -- would you mind using [`printf` instead of `echo -e`](http://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo)?  We can merge it as soon as you change that.  Thanks!

@mark-rushakoff / @d

**Comment by trastle** [comment]
Heya, forgot about this. 
I have made the change, I agree the result is better but it is a little less readable.

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
       password: c1oudc0w
 - name: postgresql_service_node
```
Comment (ruthie) [inline README.md:67]: This line should be mysql_service_node now also, no?

**Comment by ruthie** [comment]
Hi,

Upon looking at the Readme as a whole, it looks like your revisions are a step in the right direction, but leave some parts of the document referring to postgresql while others mention mysql. If we update this, we would like it to be consistent.  We'd welcome a pull request which changes the whole example to mysql, but we also have a task in our backlog to update this now.

Thank you for bringing this to our attention.

@ruthie @charliebevis

**Pull request merged**